components/ContentBar/ContentBar.jsx

```diff
@@ -56,8 +56,10 @@ export default class ContentBar extends React.Component {
           submitButtonText={this.props.submitButtonText}
           showToolbar={this.props.showModalToolbar}
           clearAfterSave={true}
+          loggedInUser={this.props.loggedInUser}
```

Comment:
What is this used for? Won't the user be unable to use if they're not logged in?

Reply:
It's to pass the logged in user through ContentBar to the CreateResourceModal to the Post to the Poll. The poll needs to know the user ID and roles for some client side checks such as whether to allow option deletion (depends on whether the logged in user is the option creator or whether the logged in user is an admin) and whether the logged in user has already voted for a particular option.